samples/data/busiest-europe-airports-2021.json

```diff
@@ -0,0 +1,1202 @@
+[
+  {
+    "type": "large_airport",
+    "name": "Berlin Brandenburg Airport",
+    "total_passengers": 9947006,
+    "lat": 52.362247,
+    "lon": 13.500672,
+    "country_name": "Germany",
+    "iso_country": "DE",
+    "region_name": "Brandenburg",
+    "iso_region": "DE-BR",
+    "municipality": "Berlin"
```

Comment:
`name`, `total_passengers`, `lat` and `lon` properties are necessary.
What about the rest? Will we need them in the future?

Reply:
I left this data with extra properties because maybe in the future someone could use it.

Reply:
Personally, I would get rid of it. Usually we don't add unnecessary information that is not used in the demo itself